.vscode/tasks.json

```
@@ -7,31 +7,22 @@
 			"command": "host start",
 			"problemMatcher": "$func-python-watch",
 			"isBackground": true,
-			"dependsOn": "pip install (functions)",
+			"dependsOn": "poetry install",
 			"options": {
-				"cwd": "${workspaceFolder}/code/backend/batch"
+				"cwd": "${workspaceFolder}"
```

```suggestion
				"cwd": "${workspaceFolder}/code/backend/batch"
```

If you add this back, and then within VS Code you change the Python  interpreter to the one from Poetry, then it works 😄

---

Will check this and raise a new PR. Merging this PR for now.